Packet++/src/TcpReassembly.cpp

```diff
@@ -132,20 +132,28 @@ void TcpReassembly::reassemblePacket(Packet& tcpData)
 	else if (tcpData.isPacketOfType(IPv6))
 		ipLayer = (Layer*)tcpData.getLayerOfType<IPv6Layer>();
 
-	if (ipLayer == NULL)
+	if (ipLayer == NULL) 
+	{
+		status = Error_NonIpPacket;

```

Comment:
I'm not sure we should call it an "error", it's just a simple case where the packet is not of type IPv4 or IPv6